### ISSUE TITLE
 HttpException: Failed Forbidden Flutter 

### ISSUE THREAD

**Issue by DevDropdown** [0maru/twitter_login#108]
I am using flutter package - twitter_login: ^4.2.3 in my app 
current testing in android 

I am getting error message after redirecting to app from twitter auth page

here is my code - 
![image](https://user-images.githubusercontent.com/108649472/177267631-24f17918-e913-4af8-9c12-e1626e4a7e40.png)
![image (1)](https://user-images.githubusercontent.com/108649472/177267680-e6a3de27-41dd-4fe6-a4f3-b8a367063275.png)

I also  added intent in android manifest

**Comment by 0maru** [comment]
Make sure you have Internet permissions in android/app/src/main/AndroidManifest.xml

```
<uses-permission android:name="android.permission.INTERNET"/>
```

**Comment by DevDropdown** [comment]
Issue Resolved by using - twitter_login: ^4.1.0-dev
